pom.xml

```diff
@@ -99,7 +99,7 @@
             <dependency>
                 <groupId>com.google.guava</groupId>
                 <artifactId>guava</artifactId>
-                <version>18.0</version>
+                <version>25.1-jre</version>
```

Comment:
Can you remind me why don't we go with version 30?

Reply:
Because Hadoop version 2 is only compatible with Guava up to version 25.